### PR TITLE
Sqlite updates

### PR DESCRIPTION
**Pull request by joewagner**

### CODE REVIEW

**Comment by joewagner** [comment]
@asutula I added some things on top of your changes to get the tests passing with the sqlite constraints.

File: up.ts [hunk 1]
```diff
@@ -114,16 +116,26 @@ const start = async function () {
...
     await deployRegistry.status();
 
+    // Add the registry address to the Validator config
+    const configFilePath = join(VALIDATOR_DIR, 'local/api/config.json');
+    const validatorConfig = JSON.parse(
+        await Deno.readTextFile(configFilePath)
+    );
+    validatorConfig.Chains[0].Registry.ContractAddress = '0xe7f1725e7734ce288f8367e1bb143e90bb3f0512';
+
+    await Deno.writeTextFile(configFilePath, JSON.stringify(validatorConfig, null, 2));
```
Comment (joewagner) [inline up.ts:126]: Figured we could save some future headaches if we automate setting the registry address in the validator config

File: test/test.js [hunk 2]
```diff
@@ -51,13 +52,13 @@ describe('Validator, Chain, and SDK work end to end', function () {
         const tableland = await getTableland(signer);
 
         const prefix = 'test_create_write';
-        const receipt = await tableland.create('key TEXT, val TEXT', prefix);
+        const receipt = await tableland.create('keyy TEXT, val TEXT', prefix);
```
Comment (joewagner) [inline test/test.js:55]: `key` is a reserved word in sqlite

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "test-js": "node --experimental-fetch --experimental-vm-modules node_modules/.bin/jest --verbose",
-    "up": "HARDHAT_DIR=../evm-tableland VALIDATOR_DIR=../go-tableland/local deno run --allow-run --allow-env up.ts"
+    "up": "HARDHAT_DIR=../evm-tableland VALIDATOR_DIR=../go-tableland/docker deno run --allow-run --allow-env --allow-read --allow-write up.ts"
```
Comment (joewagner) [inline package.json:4]: This enable reading and writing to the validator config file

File: .env_validator [hunk 4]
```diff
@@ -0,0 +1,2 @@
+ALCHEMY_KOVAN_API_KEY=XH1DyqcqyVdjnfclObUOc71oGCbAttig
```
Comment (sanderpick) [inline .env_validator:1]: this shouldn't be committed, right?

Comment (joewagner) [reply .env_validator:1]: good call

**Comment by joewagner** [comment]
@sanderpick I fixed that issue with the .env file, and rerequested review.  Let me know if you see any other issues

File: up.ts (hunk 1 above)
Comment (asutula) [reply up.ts:146]: nice

**Review by asutula — APPROVED** [review]

**Pull request merged**